reduction/Imaging_instructions.txt

```diff
@@ -72,3 +72,40 @@ of the important scripts (i.e., this directory:
 https://github.com/ALMA-IMF/reduction/tree/master/reduction).
 
 When you create mask region files, they belong in $ALMAIMF_ROOTDIR/clean_regions/.
+
+
+Additional tricks of cleaning in selfcal
+=============================================
+Be conservative in making the first image before the first iteration of the selfcal.
+That is, clean relatively relatively shallow probably by setting the threshold to be
+4 times rms or a bit greater. Otherwise, some artefacts may be introduced into the final
+image. In the later iterations (e.g., from iter2, 3, ...,) of selfcal, the threshold can be
+constrained down to 2 times rms.
+
+Scales in band3 could be set larger than those in the band 6, which is demonstrated for the 
+source G353.41, for example, scales = [0, 3, 9, 27] in band3 but scales = [0, 3, 9] in band6. 
+Note that the scales are ought to be in a reasonable range, i.e., smaller than
+or comparable to the size the largest mask, otherwise, the final image could be distorted.
+
+In each selfcal, the threshold and the region mask could be changeable due to the rms varying
+with each selfcal. So, it is recommend to change them following your previous criteria 
+on the threshold and region mask. 
+
+The mask can be made with different criteria for dense and
+diffuse regions. For example, the dense regions can be masked out with a contour of 4*rms, but
+the diffuse regions can be done with 3*rms. Checking the residual image from the right previous selfcal
+is always useful to make a good mask for a good cleaning that could result in a more flatter
+residual image in the next iteration of selfcal.
+
+After each selfcal, it is recommended to count the immediate maximum flux density , and rms, 
+to calculate the dynamical rage, and finally to decide if further selfcals are needed by check
+how the maximum flux density runs (up for more selfcals, and down for no previous one, and further
+selfcals).
```

Comment:
I'm not sure these criteria are that useful for deciding when to do additional self-calibration.  In principle, one could do self-calibration iterations until there is little to no change in flux or RMS.  It might be more helpful to show an example of your or my printout from the google doc, where we show these measurements for each iteration

Reply:
I agree with your concern after thumbing through the literature entitled with Advanced Gain Calibration Techniques in Radio Interferometry. From it, I learned that excepted for the above mentioned criteria, the integrated flux density, and even the ratio between it and the rms deserve to be checked after each iteration. In addition, I may have a chance to post the comparison of images on some parameters, very similar to what I did on the weblog at the previous meeting.